src/Spice86.Core/Emulator/InterruptHandlers/Dos/Ems/EmsSubFunctions.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Spice86.Core.Emulator.InterruptHandlers.Dos.Ems;
+
+internal static class EmsSubFunctions {
+    public const byte MapUnmapPages = 0x00;
```

Comment:
I would put that in the handler directly

Reply:
Will do.

Reply:
Done.